blocks/identity-block/themes/news.json

```diff
@@ -75,5 +78,75 @@
 			},
 			"desktop": {}
 		}
+	},
+	"sign-up": {
+		"styles": {
+			"default": {
+				"inline-size": "calc(100% - var(--global-spacing-6))",
+				"margin": "auto",
+				"font-family": "var(--font-family-primary)",
+				"max-inline-size": "none",
+				"components": {
+					"button": {
+						"font-size": "var(--global-font-size-4)"
+					},
+					"heading": {
+						"border-block-end-color": "var(--border-color)",
+						"border-block-end-style": "var(--global-border-style-1)",
+						"border-block-end-width": "var(--global-border-width-1)",
+						"font-size": "var(--global-font-size-9)",
+						"margin-block-end": "var(--global-spacing-4)",
+						"margin-bottom": "var(--global-spacing-4)",
```

Comment:
This is temporary as another block is styling this component so it effectively had `margin-bottom: 0` as a property, overwriting the `margin-block-end` property.